packages/core/contracts/common/implementation/AncillaryData.sol

```diff
@@ -74,7 +74,7 @@ library AncillaryData {
         bytes memory key,
         address value
     ) internal pure returns (bytes memory) {
-        bytes memory prefix = _appendKey(currentAncillaryData, key);
+        bytes memory prefix = constructPrefix(currentAncillaryData, key);
```

Comment:
- The [`_appendKey` function](https://github.com/UMAprotocol/protocol/blob/a3bf46270787cbaae4ed2218f064b1217c153a50/packages/core/contracts/common/implementation/AncillaryData.sol#L103) is confusingly named, because it doesn't append anything. The name `_constructPrefix` or similar would be clearer.